prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -2262,6 +2262,7 @@ int rxr_pkt_proc_compare_rta(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry)
 	cmp_data = src_data + rx_entry->total_len;
 	offset = 0;
 
+#ifdef HAVE___INT128
```

Comment:
Can you add some comments in the commit message about this change? From the PR title, I know it is to fix a compilation issue. but it is not clear from the the commit message.

Reply:
Done